platform/core/PolyPodCoreSwift/Sources/PolyPodCoreSwift/Core.swift

```diff
@@ -74,9 +74,9 @@ public final class Core {
             }
         }
     }
-    
+
```

Comment:
Uf, I remember about these spaces, do I remember that... Please check [this](https://github.com/polypoly-eu/polyPod/pull/757#pullrequestreview-911738735) and [this](https://github.com/polypoly-eu/polyPod/pull/757) and [this](https://github.com/polypoly-eu/polyPod/pull/757#issuecomment-1070769296) and finally [this one](https://github.com/polypoly-eu/polyPod/pull/757#issuecomment-1070848319). Long story short: leave whitespace alone.